### PR TITLE
Optimizations and formatting changes.

### PR DESCRIPTION
**Pull request by S3cBar0n**
Simplified some of the comparisons for wishMe(). There was an instance where to separate functions were performing the same code to retrieve a secret, I off loaded that into its own function.

Some of the smaller functions were creating unneeded variables, especially since they were being returned right away, so I simplied those further by directly returning the result instead of the redundant variable.

Commented out splitWords() since it was not being used in the code.

Other minor formatting changes and optimizations

### CODE REVIEW

File: era_ai.py [hunk 1]
```diff
@@ -98,39 +106,19 @@ def fetchNameEmail():
...
-    name1ListLower = list(map(lambda x:x.lower(), name1List))
-    name1ListLowerSplit = list(map(lambda x: x.split(), name1ListLower))
-    return name1ListLowerSplit
+    name1ListLower = list(map(lambda x: x.lower(), name1List))
+    return list(map(lambda x: x.split(), name1ListLower))
+
 
 def fetchNamePhoneNo():
     """Fetches name and their phone numbers from google contacts"""
-    creds = None
```
Comment (aritraroy24) [inline era_ai.py:114]: Others are ok...thank you for prettifying the code...but why did you delete this code block?

Comment (S3cBar0n) [reply era_ai.py:114]: You had two functions that contained that code block, which made it a bit redundant, I offloaded that block into its own fetchSecret() function which returns your secret, which is then assigned to your secret variable that you use for authentication in those functions you need the secret in.

It was just a minor optimization.

Comment (S3cBar0n) [reply era_ai.py:114]: Let me know if you have any questions

Comment (aritraroy24) [reply era_ai.py:114]: Yeah, got it...but, if I delete this code block, the code for checking the existence of the token.pickle file and also the credentials will also be gone...just delete the other function having only the code of opening the pickle file...I think that will do the job of merging into one function, keeping all the necessary codes.

Comment (S3cBar0n) [reply era_ai.py:114]: Hey @aritraroy24!

They are not gone, once your function that needs the secret it will call the fetchSecret() function which will check for the token.pickle file and return the credentials to the original function calling the fetchSecret() function.

Your checks are all still in place. 

If I am not understanding what you're saying, please elaborate.

Comment (aritraroy24) [reply era_ai.py:114]: @S3cBar0n Yeah...got it...thanks😊
